charts/kueue/templates/components/manager/manager.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "kueue.fullname" . }}-controller-manager
+  namespace: '{{ .Release.Namespace }}'
```

Comment:
```suggestion
  namespace: {{ include "kueue.namespace" . }}
```

Reply:
@ArangoGutierrez 
Thanks for your comment!

Is `{{ include "kueue.namespace" . }}` more recommended than `{{ Release.Namespace }}` ?

I just referenced scheduler-plugins repository.
https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/manifests/install/charts/as-a-second-scheduler

Reply:
I have read [helm's document](https://helm.sh/docs/chart_template_guide/builtin_objects/#helm) and [helm repo](https://github.com/helm/helm/tree/main), but I couldn't find a specific reason for using `{{ include "kueue.namespace" . }}` in the chart. 

If there is a specific reason for this, please let me know and I will fix it.
For improving this PR's readability, I will close same comments (if necessary, I will fix all parts of your comments).

Thanks,